sycl/include/CL/sycl/detail/image_accessor_util.hpp

```diff
@@ -0,0 +1,691 @@
+#ifndef __SYCL_DEVICE_ONLY__
```

Comment:
Needs the generic LLVM file header with the Apache license

Reply:
done.